runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContent.java

```diff
@@ -57,7 +59,7 @@
     private static Class<? extends Annotation> javascriptInterfaceClass = null;
 
     private ContentViewCore mContentViewCore;
-    private ContentView mContentView;
+    private XWalkContentView mContentView;
```

Comment:
I didn't see why XWalkContentView  is required. ContentView.onCreateInputConnection is public.